include/controller.h

```diff
@@ -44,6 +44,19 @@
 #define ACCESSORY_TRANSFERPAK    4
 /** @} */
 
+/**
+ * @name EEPROM Probe Values
+ * @see #eeprom_present
+ * @{
+ */
+/** @brief No EEPROM present */
+#define EEPROM_NONE          0
+/** @brief 4 kilobit (64-block) EEPROM present */
+#define EEPROM_4k            1
+/** @brief 16 kilobit (256-block) EEPROM present */
+#define EEPROM_16k           2
+/** @} */
```

Comment:
Minor nit: we should keep the macros uppercase, so this should be `EEPROM_4K` and `EERPOM_16K`.